### ISSUE TITLE
Opening up your projects in the editor can make them suddenly autosave

### ISSUE THREAD

**Issue by DNin01** [LLK/scratch-gui#8385]
### Expected Behavior

They should open without saving immediately.

### Actual Behavior

Clicking "See inside" from My Stuff to open projects (or just opening them in the editor from a link or even refreshing the page while in the editor) often makes Scratch immediately save them (I can see "Saving project..." then "Project saved" after the project loads).

I don't know if maybe part of Scratch thinks you're starting a new project and that's why it autosaves?

### Steps to Reproduce

Any of the following:
- Clicking "See inside" from My Stuff
- Refreshing the page in the editor
- Opening a link to your project with `/editor`, like scratch.mit.edu/projects/123456789/**editor**

It might not always do it the first time.

I don't have any videos at the moment.

### System Details

Windows 10 with Edge 101.0.4951.64

**Comment by mxmou** [comment]
This can also change the project's thumbnail and the last modified date.

**Comment by griffpatch** [comment]
Indeed, i cannot visit any of my projects on mobile because if I view any,
even without looking inside, the projects autosave and I lose my thumbnail
